STYLE.md

```diff
@@ -96,3 +103,14 @@ Format anyhow errors with `{:#}` to print all the contexts like this:
 ```
 error!(context, "Failed to set selfavatar timestamp: {err:#}.");
 ```
+
+## Documentation comments
+
+All public modules, methods and fields should be documented.
+This is checked by [`missing_docs`](https://doc.rust-lang.org/rustdoc/lints.html#missing_docs) lint.
+
+Private items do not have to be documented,
+but `cargo doc --document-private-items`
```

Comment:
Is this sentence finished? Should `cargo doc --document-private-items` be run and when if so?